src/pages/workspace/WorkspacePeoplePage.js

```diff
@@ -0,0 +1,328 @@
+                        <View style={[styles.peopleRowCell, styles.flex4]}>
+                            <View style={[styles.avatarWithName]}>
+                                <Avatar
+                                    imageStyles={[styles.mr2]}
+                                    source={item.avatar}
+                                />
+                                <View style={[styles.dFlex, styles.flexColumn, styles.justifyContentCenter]}>
+                                    <Text style={[styles.textStrong, styles.peopleMobileAssigneeText]}>
+                                        {item.displayName}
+                                    </Text>
+                                    <Text style={[styles.textLabel, styles.colorMuted, styles.peopleMobileAssigneeText]}>
+                                        {item.login}
+                                    </Text>
+                                </View>
+                            </View>
+                        </View>
+                    ) : (
+                        <Fragment>
+                            <View style={[styles.peopleRowCell, styles.flex4]}>
+                                <View style={[styles.avatarWithName]}>
+                                    <Avatar
+                                        imageStyles={[styles.mr2]}
+                                        source={item.avatar}
+                                    />
+                                    <Text style={[styles.textStrong]}>
+                                        {item.displayName}
+                                    </Text>
+                                </View>
+                            </View>
+                            <View style={[styles.peopleRowCell, styles.flex4]}>
+                                <Text style={[styles.textLabel, styles.colorMuted]}>
+                                    {item.login}
+                                </Text>
+                            </View>
+                        </Fragment>
+                    )
+                }
+                <View style={[styles.peopleRowCell, styles.flex1, styles.peopleBadgesContainer]}>
+                    {
+                        this.props.session.email === item.login && (
+                            <View style={[styles.peopleBadge]}>
+                                <Text style={[styles.peopleBadgeText]}>
+                                    Admin
+                                </Text>
+                            </View>
+                        )
+                    }
+                </View>
+            </View>
+        );
+    }
+
+    renderHeader() {
+        return (
+            <View style={[styles.peopleRow, styles.peopleHeaderRow]}>
+                <View style={[styles.peopleRowCell, styles.peopleCheckbox]}>
+                    <Checkbox
+                        isChecked={this.state.selectedEmployees.length === this.props.policy.employeeList.length}
```

Comment:
Let's use something like `this.state.shouldSelectAllEmployees`? While the employeeList being equal in length to the selectedEmployees is the end result we are looking for and we _could_ use it to determine which way to toggle we'll need to do this same check every time.

Reply:
I was thinking about this, but then I think it would end up in quite clumsy `toggleUser` function.

Let me know what you think and what should we choose. The function would have to look like this:

```
toggleUser(login) {
        if (login === 'ALL') {
            this.setState(prevState => ({
                shouldSelectAllEmployees: !prevState.shouldSelectAllEmployees,
                selectedEmployees: this.props.policy.employeeList.length !== prevState.selectedEmployees.length 
                    ? this.props.policy.employeeList 
                    : [],
            }));
            return;
        }

        this.setState(prevState => ({
            shouldSelectAllEmployees: ((!_.contains(prevState.selectedEmployees, login) && prevState.selectedEmployees.length === this.props.policy.employeeList.length - 1)
                ? true
                : false,
            selectedEmployees: _.contains(prevState.selectedEmployees, login)
                ? _.without(prevState.selectedEmployees, login)
                : [...prevState.selectedEmployees, login],
        }));
    }
```

The condition there is quite hard to read. We need to check if the `login` is in `prevState` and also if it is the last one 'unchecked'. We could write it in a longer more readable way but I am not sure if the advantages of having the `shouldSelectAllEmployees` state key are outweighing the worse readability. 

Let me know what you think!